schedule/containers/engines_and_tools_docker.yaml

```diff
@@ -1,29 +1,36 @@
-name:           extra_tests_textmode_containers
```

Comment:
i kept the name as it was. i am not sure if it was kept like this for some reason

Reply:
I don't think so. cc: @jlausuch ?

Reply:
i changed it then

Reply:
I think I forgot to update it.